### PR TITLE
[processor/attributes] Clarify logs are supported by this processor

### PR DESCRIPTION
**Pull request by mstumpfx**
**Description:** Minor readme update for

**Link to tracking Issue:** n/a

**Testing:** n/a

**Documentation:** Clarification in README that this processor applies actions and filtering for log resources. It seems it was originally written for spans but not cleaned up when log support was added.

### CODE REVIEW

File: processor/attributesprocessor/README.md [hunk 1]
```diff
@@ -105,11 +106,11 @@ processors:
 Refer to [config.yaml](./testdata/config.yaml) for detailed
 examples on using the processor.
 
-## Include/Exclude Spans
+## Include/Exclude Filtering
 
 The [attribute processor](README.md) and the [span processor](../spanprocessor/README.md) expose
-the option to provide a set of properties of a span to match against to determine
-if the span should be included or excluded from the processor. To configure
+the option to provide a set of properties of a span/logs to match against to determine
```
Comment (dmitryax) [inline processor/attributesprocessor/README.md:112]:
```suggestion
the option to provide a set of properties of a span or log record to match against to determine
```
?

File: processor/attributesprocessor/README.md [hunk 2]
```diff
@@ -105,11 +106,11 @@ processors:
 Refer to [config.yaml](./testdata/config.yaml) for detailed
 examples on using the processor.
 
-## Include/Exclude Spans
+## Include/Exclude Filtering
 
 The [attribute processor](README.md) and the [span processor](../spanprocessor/README.md) expose
```
Comment (dmitryax) [inline processor/attributesprocessor/README.md:111]: I'd say we should remove span processor link from here to avoid confusion

Comment (mstumpfx) [reply processor/attributesprocessor/README.md]: Agreed. I also updated the example config with the rest of the supported fields.

File: processor/attributesprocessor/README.md [hunk 3]
```diff
@@ -105,19 +106,22 @@ processors:
...
 
-The [attribute processor](README.md) and the [span processor](../spanprocessor/README.md) expose
-the option to provide a set of properties of a span to match against to determine
-if the span should be included or excluded from the processor. To configure
-this option, under `include` and/or `exclude` at least `match_type` and one of
-`services`, `span_names` or `attributes` is required.
+The [attribute processor](README.md) exposes
+an option to provide a set of properties of a span or log record to match against to determine
+if the span/log should be included or excluded from the processor. To configure
+this option, under `include` and/or `exclude` at least `match_type` and one of of the following
```
Comment (dmitryax) [inline processor/attributesprocessor/README.md:114]:
```suggestion
this option, under `include` and/or `exclude` at least `match_type` and one of the following
```

File: processor/attributesprocessor/README.md [hunk 4]
```diff
@@ -135,14 +139,26 @@ are checked before the `exclude` properties.
...
       # This is an optional field.
       services: [<item1>, ..., <itemN>]
 
+      # resources specify an array of items to match the resources against.
+      # A match occurs if the span/log resources matches at least one of the items.
+      resources: [<item1>, ..., <itemN>]
+
+      # libraries specify an array of items to match the implementation library against.
+      # A match occurs if the span/log implementation library matches at least one of the items.
+      resources: [<item1>, ..., <itemN>]
```
Comment (dmitryax) [inline processor/attributesprocessor/README.md:152]: libraries?

Comment (mstumpfx) [reply processor/attributesprocessor/README.md]: Yep, thanks!

**Review by dmitryax — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**